modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -0,0 +1,70 @@
+
+This way, you can use the timestamp in `buildConfig.status.imageChangeTriggers` to identify the most recent build. Then you can use the name and namespace values of the image stream that triggered this build to find the corresponding image change trigger in the `buildConfig.spec.triggers`.
+
+
+== Image change trigger elements in the specification
+
+In your build configuration specification, `buildConfig.spec.triggers` is an array of build trigger policies, `BuildTriggerPolicy`.
+
+Each `BuildTriggerPolicy` has a `type` field and set of pointers. The `type` can be set to only one pointer. For image change triggers, the `type` can take different forms, depending on how you defined the triggers:
+
+* `buildConfig.spec.triggers[i].kind = ImageChange`
+* `buildConfig.spec.triggers[i].imageChange`, deprecated, whose value is the JSON or YAML of an `ImageChangeTrigger`
+* `buildConfig.spec.triggers[i].imageChange.from`, deprecated, whose value is an object reference
+
+The following example shows the relative location of the elements mentioned in the preceding list and omits unrelated elements, such as `name`, `source`, and `strategy`.
+
+.Example `BuildConfig.spec`
+[source,yaml]
+----
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+spec:
+  triggers:
+  - imageChange:
+      from:
+        kind: ImageStreamTag
+        name: <1>
+        namespace: <2>
+    type: ImageChange
+----
+<1> The name of an image stream, such as `input:latest`.
+<2> A namespace, such as `my-namespace`.
+
+== Image change trigger elements in the status
+
+In your build configuration status, `buildConfig.status.imageChangeTriggers` is an array of `ImageChangeTriggerStatus` elements. Each `ImageChangeTriggerStatus` element includes the `from`, `lastTriggeredImageID`, and `lastTriggerTime` elements shown in the following example. This example omits elements that are not related to image change triggers.
+
+.Example `BuildConfig.status`
+[source,yaml]
+----
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+status:
+  imageChangeTriggers:
+     - from:
+          name: <1>
+          namespace: <2>
+       lastTriggeredImageID: <3>
+       lastTriggerTime: <4>
+----
+<1> The name of an image stream, such as `input:latest`.
+<2> A namespace, such as `my-namespace`.
+<3> The SHA or ID of the `ImageStreamTag` when a build started. Its value is updated each time a build is started, even if this `ImageStreamTag` is not the reason the build started.
+<4> The last time this particular `ImageStreamTag` triggered a build to start. Its value is only updated when this trigger specifically started a Build.
+
+The `ImageStreamTag` that has the most recent `lastTriggerTime` triggered the most recent build. You can use its `name` and `namespace` to correlate the `ImageStreamTag` one of the image change triggers you defined in the `buildConfig.spec.triggers`.
```

Comment:
Close.  Some slight modifications:

The `ImageChangeTriggerStatus` that has the most recent `lastTriggerTime` triggered the most recent build. You can use the `name` and `namespace` to correlate to the `ImageStreamTag` of one of the image change triggers you defined in the `buildConfig.spec.triggers`.

Reply:
Updated. Thanks!